src/components/Modal/wrappers/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react"
```

Comment:
Can we also export a `Title` component that will have the styles which the previous default header did? I suppose only `font-size` and `line-height` should be required, this help maintain consitency and avoid duplications...